src/_posts/databases/mysql/2000-01-01-connecting.md

```diff
@@ -91,13 +91,9 @@ If you want to enforce TLS connections to your database so that TLS is
 not only available, but **mandatory**:
 
 1. From your web browser, [open your database dashboard]({% post_url databases/mysql/2000-01-01-getting-started %}#accessing-the-scalingo-for-mysql-dashboard)
-2. Select the **Overview** tab
-3. Locate the **TLS/Internet Access** block
-4. Toggle the corresponding option:
-
-{% assign img_url = "https://cdn.scalingo.com/documentation/screenshot_database_mongo_force_tls.png" %}
-{% include mdl_img.html %}
-
+2. Select the **Settings** tab
```

Comment:
I was thinking about doing this :

1. From your web browser, [open your database dashboard]({% post_url databases/postgresql/2000-01-01-getting-started %}#accessing-the-scalingo-for-postgresql-dashboard)
2. Select the **Settings** tab
2. Click on the **Internet Access** menu on the left
3. Locate the **Force TLS connections** block
4. Toggle the corresponding option:

With the screenshot

Reply:
WDYT

Reply:
I rather use "enable", since that's the name of the button